LICENSE.md

```diff
@@ -0,0 +1,97 @@
+﻿OPENBARE LICENTIE VAN DE EUROPESE UNIE v. 1.2. 

```

Comment:
Shouldn't we use the English version of EUPL?

Reply:
Changed to English